### DISCUSSION TITLE
outpus/inputs rerun if directory is missing

### DISCUSSION THREAD

**Discussion by gbkwiatt** [vercel/turborepo#1985] [Help] [answered]
I am trying to configure task, that creates directory and copy files into it. 
However I want to rerun that task if the directory is missing. 

e.g. 
I have task `mkdir -p ./src/site/static/icons && cp -r ../../node_modules/lucide-static/icons/* ./src/site/static/icons`
inside each workspace.  which are in `apps` directory 

then in main turbo.json 
```json
   "icons:copy":{
      "inputs": ["src/static/icons/**"]
    },
```
so if I delete directory it does not rerun the job and says it's cached. 

Side question. should it inputs path be relative to workspace or to whole repo like `apps/**/src/static/icons` ?

**Comment by anthonyshew** [comment, marked as answer]
You need an `outputs` key so that `turbo` knows what to cache.

https://turbo.build/repo/docs/reference/configuration#outputs
